content/blog/2024/10/solus-4-6-released/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Solus 4.6 Released"
+author: "david"
+categories:
+  - news
+  - releases
+date: 2024-10-14
+featuredimage: "/solus-4.6-featured.jpg"
+url: "/2024/10/14/solus-4-6-released"
+---
+
+The Solus team is proud to announce the release of Solus 4.6 _Convergence_. This refresh brings the usual collection of Desktop Environment updates, kernel enhancements, and increased hardware support.
+
+## General
+
+### Default applications
+
+All our editions feature:
+
+- Firefox 131.0.3
+- LibreOffice 24.2.5.2
+- Thunderbird 128.3.0
+
+For audio and video multimedia playback, we offer software out-of-the-box that caters specifically to our desired experience for each edition.
+
+- Budgie and GNOME editions ship with Rhythmbox for audio playback, with the latest release of the Alternate Toolbar extension to provide a more modern user experience.
+- Budgie and GNOME ship with Celluloid for video playback.
+- XFCE ships with Parole for multimedia playback.
+- Plasma ships with Elisa for audio playback and Haruna for video playback.
+
+### Usr-merge
+
+- Over the summer, the Solus team has been tackling the [Usr-Merge conundrum](https://getsol.us/2024/09/04/usr-merge/); read Evan's excellent blog post for the full story. We have now reached the end of Stage 3 on our rollout plan: All users with an updated system now have a Usr-Merge'd system. Any users that create a fresh installation with today's ISOs will also have a merged system. We expect stage 4 to complete Soon™️.
+
+As a reminder, the Usr-Merge brings Solus in line with the compatibility efforts being spearheaded by Fedora and other distributions. More urgently, *systemd* requires the filesystem be merged for version 255.
+
+### Experimental Software Centers
+
+We continue to ship *solus-sc* (Solus Software Center) as the default graphical package manager, but close watchers of Solus development know that we have had experimental support for GNOME Software and KDE Discover included in our repository for many months now.
+
+These software centers let users update software distributed by the Solus repositories, as before, *and* install and manage Flatpaks for third party packages. Flathub is configured as a Flatpak remote by default.
```

Comment:
```suggestion
These software centers let users update software distributed by the Solus repositories, as before, *and* install and manage Flatpaks for third-party packages. Flathub is configured as a Flatpak remote by default.
```

Merriam-Webster (the preferred dictionary according to the Help Center style guide) hyphenates the adjective form (and it's also written that way further down the text)